VideoWeb/VideoWeb/Mappings/ConferenceForUserResponseMapper.cs

```diff
@@ -49,61 +53,11 @@ public ConferenceForUserResponse MapConferenceSummaryToResponseModel(ConferenceS
             conferenceForUserResponse.CaseType = conference.Case_type;
             conferenceForUserResponse.ScheduledDateTime = conference.Scheduled_date_time;
             conferenceForUserResponse.ScheduledDuration = conference.Scheduled_duration;
-            conferenceForUserResponse.Status = MapConferenceStatus(conference.Status);
+            conferenceForUserResponse.Status = Enum.Parse<ConferenceStatus>(conference.Status.ToString());
```

Comment:
Should try parse this else ex will bubble up

Reply:
It should fail, the default would be the None value otherwise.  If a new enum, or unrecognised string, is sent from Video API this app should fail rather than hide it.